arch/arm/include/arm/irq.h

```diff
@@ -240,22 +232,22 @@ int up_cpu_index(void) noinstrument_function;
 #endif /* CONFIG_ARCH_HAVE_MULTICPU */
 
 noinstrument_function
-static inline_function uint32_t *up_current_regs(void)
+static inline_function void arm_set_irq_flag(bool flag)
```

Comment:
```suggestion
static inline_function void arm_set_interrupt_context(bool flag)
```
move to arm_internal.h